packages/env/lib/config/parse-config.js

```diff
@@ -191,6 +193,7 @@ async function getDefaultConfig(
 			WP_SITEURL: 'http://localhost',
 			WP_HOME: 'http://localhost',
 		},
+		afterSetup: '',
```

Comment:
Could we just leave it undefined? I think we have some options like that, such as phpVersion, which are mostly optional and only need to be used if they're set. I think it's slightly clearer that way

Reply:
I suppose that's true! I'll take a look at making that change.